cmd/object-api-utils.go

```diff
@@ -1206,6 +1206,19 @@ func hasSpaceFor(di []*DiskInfo, size int64) (bool, error) {
 	}
 
 	if nDisks < len(di)/2 || nDisks <= 0 {
+		var errs []string
```

Comment:
I think we can use the new errors.Join() API (go 1.20), I think it will be easier.

Reply:
@vadmeste errors.Join(errs ....error). 
```go
type DiskInfo struct {
	...
	Error      string // carries the error over the network
}
```
That's string type. So I think it's not easier.